### PR TITLE
feat: Activity Stream Implement placeholder for activity stream - MEED-941

### PR DESCRIPTION
**Pull request by ArfaouiMohamedAmine**
this change is going to `alter` the old `placeholder` for `space activity stream` with a new one .

### CODE REVIEW

File: webapp/portlet/src/main/webapp/vue-apps/activity-stream/components/empty-stream/ActivityStreamEmptyMessageSpace.vue [hunk 1]
```diff
@@ -1,40 +1,126 @@
...
+  beforeDestroy() {
+    document.removeEventListener(this.emptyActivityStreamActionName, this.clickOnSpacesActivityStreamEmptyActionLink);
+  },
   computed: {
     welcomeTitle() {
       return this.$t && this.$t('UIActivity.label.Welcome_Space', {
         'space name': `<strong>${eXo.env.portal.spaceDisplayName}</strong>`,
       });
     },
+    firstFeatureMessage() {
```
Comment (boubaker) [inline webapp/portlet/src/main/webapp/vue-apps/activity-stream/components/empty-stream/ActivityStreamEmptyMessageSpace.vue]:
```suggestion
    firstFeatureMessage() {
```
please use more explicit names for variables only if really it's about just an order (first, second ...)
here it should be `emptyStreamForSpacesThatUserManage` by example. Same for others.

File: webapp/portlet/src/main/webapp/vue-apps/activity-stream/components/empty-stream/ActivityStreamEmptyMessageSpace.vue [hunk 2]
```diff
@@ -1,40 +1,126 @@
...
+      return this.$t && this.$t('UIActivity.label.empty_space_stream_manage_community');
```
Comment (boubaker) [inline webapp/portlet/src/main/webapp/vue-apps/activity-stream/components/empty-stream/ActivityStreamEmptyMessageSpace.vue]: `this.$t` is always truthy

File: webapp/portlet/src/main/webapp/vue-apps/activity-stream/components/empty-stream/ActivityStreamEmptyMessageSpace.vue [hunk 3]
```diff
@@ -1,40 +1,126 @@
...
+        max-width="500"
+        flat
+        class="mx-auto my-8">
+        <v-row>
+          <activity-stream-empty-message-space-icons-column
+            :icon-size="iconSize"
+            :icon-ref="userGearIcon"
+            :info-message="firstFeatureMessage" />
+          <activity-stream-empty-message-space-icons-column
+            icon-color="secondary"
```
Comment (boubaker) [inline webapp/portlet/src/main/webapp/vue-apps/activity-stream/components/empty-stream/ActivityStreamEmptyMessageSpace.vue]: please reorder attributes

File: webapp/portlet/src/main/webapp/vue-apps/activity-stream/components/empty-stream/ActivityStreamEmptyMessageSpace.vue [hunk 4]
```diff
@@ -1,40 +1,126 @@
...
-      loading="lazy">
-    <div class="mx-4 mb-4">
+    <div class="mx-4 my-4">
       <p v-sanitized-html="welcomeTitle"></p>
+      <v-card
+        max-width="500"
+        flat
+        class="mx-auto my-8">
+        <v-row>
+          <activity-stream-empty-message-space-icons-column
```
Comment (boubaker) [inline webapp/portlet/src/main/webapp/vue-apps/activity-stream/components/empty-stream/ActivityStreamEmptyMessageSpace.vue:26]: How to choose which message to display ? (I mean there is not  `v-if-else` block, do you display four blocks at the same time ?)

Comment (ArfaouiMohamedAmine) [reply webapp/portlet/src/main/webapp/vue-apps/activity-stream/components/empty-stream/ActivityStreamEmptyMessageSpace.vue:26]: yes I need All of them to be displayed in this way https://community.exoplatform.com/portal/rest/jcr/repository/collaboration/Groups/spaces/meeds_labers/Documents/product/ux/placeholder%20stream%20page.png?time=1666885886654

Comment (boubaker) [reply webapp/portlet/src/main/webapp/vue-apps/activity-stream/components/empty-stream/ActivityStreamEmptyMessageSpace.vue:26]: ok, thanks.

**Review by boubaker — APPROVED** [review]

**Pull request merged**